src/pages/About.styled.js

```diff
@@ -0,0 +1,87 @@
+const styles = {
+  section: {
+    py: 12,
+    px: 3,
```

Comment:
In this file : 

Do not use hardcoded colos, they should come from the theme
Do not use MUI properties - use css to ovveride them 
Do not use px values - use rem